### PR TITLE
[FX-4611] Migrate Loader

### PR DESCRIPTION
**Pull request by mkrl**
[FX-4611](https://toptal-core.atlassian.net/browse/FX-4611)

### Description

Migrates `Loader` away from MUI to pure CSS / Tailwind.

Please note: a story for controlled `Loader` now includes controls to play around with animations, it can (and will) cause a difference in the corresponding screenshot test.

### How to test

- [Temploy](https://picasso.toptal.net/fx-4611-migrate-circularprogress)
- Take a look at the code, changesets and package-json changes
- Confirm everything looks good visually, check screenshot tests
- CI should be green (apart from a new case for the screenshot)

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests ([visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md) included)

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-4611]: https://toptal-core.atlassian.net/browse/FX-4611?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by AdrianContiu — COMMENTED** [review]
I've tested visually and everything works as expected, except for a few [Happo tests](https://happo.io/a/675/p/1189/snapshot/39f7a2c11768f06d68a6fc217a7809c5/compare/14393958ff36ad88923c540e92994956) that fail, unrelated to the new controlled value story. Seems to be a spacing issue and some existing properties of [CircularProgress](https://github.com/mui/material-ui/blob/v4.x/packages/material-ui/src/CircularProgress/CircularProgress.js) previously used from `material-ui` are missing:

> circleIndeterminate: {
    
    // Some default value that looks fine waiting for the animation to kicks in.
    strokeDasharray: '80px, 200px',
    strokeDashoffset: '0px', // Add the unit to fix a Edge 16 and below bug.
  },

File: packages/picasso-tailwind/src/index.js [hunk 1]
```diff
@@ -201,6 +201,25 @@ module.exports = {
       maxHeight: ({ theme }) => ({
         ...theme('spacing'),
       }),
+      animation: {
+        'circle-spin': 'stroke-dash 1.4s ease-in-out infinite',
```
Comment (augustobmoura) [inline packages/picasso-tailwind/src/index.js:205]: Is this being used anywhere?

**Review by augustobmoura — APPROVED** [review]
Do you know what are this changes on Happo about?
Otherwise everything is working as expected

File: packages/picasso-tailwind/src/index.js (hunk 1 above)
Comment (mkrl) [reply packages/picasso-tailwind/src/index.js:205]: Yes, this is for the `<circle>` inside of the loader svg

**Comment by mkrl** [comment]
@AdrianContiu @augustobmoura I'm currently taking a look at loader style changes in Happo, those are only present in screenshots.

**Comment by augustobmoura** [comment]
> @AdrianContiu @augustobmoura I'm currently taking a look at loader style changes in Happo, those are only present in screenshots.

Maybe try rebasing the PR, I believe the problems with unrelated changes are solved now

**Pull request merged**